src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.server.ts

```diff
@@ -421,6 +452,10 @@ export const load = async ({
             }
           : undefined,
       },
+      imageGallery: {
+        images: imageGallery.filter(isNotNull),
```

Comment:
switch this to just `images: imageGallery` to see the TypeScript error I was encountering fixed by adding the filter method. If there's a cleaner way to do this in the query/type definitions above, I'd rather do that.